dispatch-service/src/main/resources/application.yml

```diff
@@ -6,6 +6,7 @@ spring:
       default:
         consumer:
           max-attempts: 3
+          ack-mode: record
```

Comment:
@coderabbitai am i right, that this is applied to all consumers and works as intended?